README.md

```diff
@@ -1,13 +1,22 @@
 # Generic Executor Service for Keptn
 
-This is a Sandbox Keptn Service that enables generic execution of bash files, Python3 and HTTP requests for individual Keptn Events 
+
+This is a Sandbox Keptn Service that enables generic execution of bash files, Python3 and HTTP requests for individual Keptn Events
 
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/generic-executor-service)
 [![Build Status](https://travis-ci.org/keptn-sandbox/generic-executor-service.svg?branch=master)](https://travis-ci.org/keptn-sandbox/generic-executor-service)
 [![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/generic-executor-service)](https://goreportcard.com/report/github.com/keptn-sandbox/generic-executor-service)
 
 This implements a generic-executor-service for Keptn.
 
+## :warning: This service is currently not maintained
+
+Instead of using generic-executor-service, we recommend to 
+
+* make use of existing [Keptn integrations](https://keptn.sh/docs/integrations/),
+* use [Keptn's built-in webhook-service](https://keptn.sh/docs/0.13.x/integrations/webhooks/) for simple HTTP webhook-based integrations,
+* or, in case of more complex setups required, try out [Keptn job-executor-service](https://github.com/keptn-contrib/job-executor-service/).
```

Comment:
As the real successor of the generic-executor is the job-executor-service, I'd mention this first. If an existing integration for a use-case would exist, then there would be no need for the generic executor.

Reply:
I agree, will re-work this. Still, HTTP based webhooks was a core functionality of generic-executor-service ;)